configure.ac

```diff
@@ -160,33 +160,26 @@ AC_ARG_ENABLE(runtime,
 # Check for enable/disable builtin PSL data
 AC_ARG_ENABLE(builtin,
   [
-  --enable-builtin[[=IDNA library]]
-      Specify the IDNA library used for built-in data generation:
-        libidn2 [[default]]: IDNA2008 library (also needs libunistring)
-        libicu: IDNA2008   UTS#46 library
-        libidn:            IDNA2003 library (also needs libunistring)
+  --enable-builtin         Generate built-in PSL data
   --disable-builtin        Do not generate built-in PSL data
   ], [
-    if test "$enableval" = "libidn2" -o "$enableval" = "yes"; then
-      enable_builtin=libidn2
-      AC_DEFINE([BUILTIN_GENERATOR_LIBIDN2], [1], [generate PSL data using libidn2])
-    elif test "$enableval" = "libicu"; then
-      enable_builtin=libicu
-      AC_DEFINE([BUILTIN_GENERATOR_LIBICU], [1], [generate PSL data using libicu])
-    elif test "$enableval" = "libidn"; then
-      enable_builtin=libidn
-      AC_DEFINE([BUILTIN_GENERATOR_LIBIDN], [1], [generate PSL data using libidn])
+    if test "$enableval" = "yes"; then
+      enable_builtin=yes
     elif test "$enableval" = "no"; then
       enable_builtin=no
     else
```

Comment:
For compatibility with existing scripts (e.g. the [PSL project](https://github.com/publicsuffix/list) uses `--enable-builtin=libicu` in one if its Makefile targets),
add just a warning for $enableval `libidn`, `libidn2` and `libicu` while setting `enable_builtin=yes` ?

Reply:
Now the older `libidn`/`libidn2`/`libicu` produce a warning : `--enable-builtin=$enableval is deprecated, use --enable-builtin (enabled by default)`.

It isn't really visible in the middle of the configure traces, but it should be fine since it doesn't break the previous behavior.